frontend/ui/widget/opdsbrowser.lua

```diff
@@ -474,6 +474,8 @@ function OPDSBrowser:downloadFile(item, format, remote_url)
     -- download to user selected directory or last opened dir
     local lastdir = G_reader_settings:readSetting("lastdir")
     local download_dir = G_reader_settings:readSetting("download_dir") or lastdir
+    item.author = item.author:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
+    item.title = item.title:gsub('[\\,%/,:,%*,%?,%",%<,%>,%|]','')
```

Comment:
PS I should probably clarify that I'm talking about something that would fit in alongside these bunch of functions: https://github.com/koreader/koreader-base/blob/c198ca05d904f61f655cf7adbe2f361098e653ac/ffi/util.lua#L101-L194